### PR TITLE
Add mobile menu to portfolio

### PR DESCRIPTION
**Pull request by Luchy-B**
- Add JS file to portfolio
- Add Mobile Menu

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -425,12 +427,64 @@ body {
...
+    color: #fff;
+  }
+
+  .closeIcon {
+    float: right;
+    font-size: 32px;
+    font-weight: 10;
+
+    /* padding-left: 280px; */
+  }
```
Comment (AlphaNtihinduka) [inline styles.css]: - [ ] I have noticed that you have the commented codes inside your CSS files. I kindly suggested that you remove them as many commented codes can create distractions 😞 .

File: styles.css [hunk 2]
```diff
@@ -425,12 +427,64 @@ body {
...
+
+  .navigation {
+    /* position: fixed; */
+    width: 100%;
+    top: 0%;
```
Comment (AlphaNtihinduka) [inline styles.css]: - [ ] I have noticed that you have the commented codes inside your CSS files. I kindly suggested that you remove them as many commented codes can create distractions 😞 .

File: styles.css [hunk 3]
```diff
@@ -467,23 +521,28 @@ body {
...
   }
 
   .header-Page {
     display: flex;
     flex-direction: column;
-    padding: 250px 260px 94px 260px;
+    justify-content: flex-start;
+    width: 50%;
+
+    /* padding: 250px 260px 94px 260px; */
```
Comment (AlphaNtihinduka) [inline styles.css]: - [ ] I have noticed that you have the commented codes inside your CSS files. I kindly suggested that you remove them as many commented codes can create distractions 😞 .

File: styles.css [hunk 4]
```diff
@@ -49,17 +49,21 @@ body {
...
   background-position: top left;
   background-repeat: no-repeat;
-  background-size: auto;
+  background-size: cover;
   overflow: hidden;
   justify-content: center;
+  width: 100%;
+
+  /* height: 100vh; */
 }
```
Comment (AlphaNtihinduka) [inline styles.css]: - [ ] I have noticed that you have the commented codes inside your CSS files. I kindly suggested that you remove them as many commented codes can create distractions 😞 .

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @Luchy-B 🙋‍♂️
Good job working on your mbile version of the portifolio. You have done the design to the best expectation and incredibly well. 👏  

However, there are some issues mostly on the CSS best practice.
 
You still need to implement few on the project but you are almost there!!! :watch: 

TO HIGHLIGHT:
- Your project is professional ✅
- Professional README file :100: 
- Your PR is professional ✅
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

File: styles.css (hunk 1 above)
Comment (rloterh) [reply styles.css]: resolved! ✔️

File: styles.css (hunk 2 above)
Comment (rloterh) [reply styles.css]: resolved! ✔️

File: styles.css (hunk 4 above)
Comment (rloterh) [reply styles.css]: resolved! ✔️

File: index.html [hunk 5]
```diff
@@ -15,16 +15,19 @@
...
-            <li><a href="#">About</li></a>
-            <li><a href="#">Contact</li></a>
-        </ul>
-    </nav>
+      <div>
+        <nav class="Hamburger" >
+          <img src="images/Hamburger2.png" alt="Hamburger" id="openHam"/>
+        </nav>
+        <nav class="navigation">
+          <ul id="cross">
```
Comment (rloterh) [inline index.html:23]: - [ ] OPTIONAL
 To enhance the UX, kindly ensure that the _hamburger_ and the _close_ button are in the same position when toggled.

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Blessing,  

Your project is complete!🏁 There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 🎯 
- Menu items jump to the corresponding part of the page when clicked ❣️
- Mobile menu icon is hidden on the desktop screen ❣️

## Optional suggestions 🎯 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

_Kindly consider my inline comments marked **[OPTIONAL]** under this review_

**You can also consider:**
- Kindly endeavor to remove the broken logo link at the topmost part of your ReadMe.
- Live demo link in your ReadMe does not match your project, kindly endeavor to update the same with the appropriate link.
- Kindly include appropriate instructions in your ReadMe on how to run your app, under the _Getting Started_ section.
- I think you should replace the subheading text Author1 with your full name in your ReadMe.
- MIT license link throws a 404-page request error when visited. Kindly endeavor to fix it.

Cheers..Happy coding! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**